src/neuromaps_nhp/graph/transforms.py

```diff
@@ -0,0 +1,187 @@
+                        all_paths[path_key] = {
+                            'path': path,
+                            'length': len(path) - 1,
+                            'direct': len(path) == 2
+                        }
+                except:
+                    continue
+    
+    # Create visualization if requested
+    if show_graph and atlases:
+        plt.figure(figsize=(12, 8))
+        
+        # Create a networkx graph
+        G = nx.Graph()
+        
+        # Add nodes
+        G.add_nodes_from(atlases)
+        
+        # Add edges from direct transforms
+        G.add_edges_from(direct_transforms)
+        
+        # Create layout
+        pos = nx.spring_layout(G, k=2, iterations=50)
+        
+        # Draw the graph
+        nx.draw_networkx_nodes(G, pos, node_color='lightblue', 
+                              node_size=1500, alpha=0.7)
+        nx.draw_networkx_edges(G, pos, edge_color='gray', alpha=0.5)
+        nx.draw_networkx_labels(G, pos, font_size=10, font_weight='bold')
+        
+        plt.title("Brain Atlas Transform Graph\n(Direct transforms shown as edges)")
+        plt.axis('off')
+        plt.tight_layout()
+        plt.show()
+    
+    # Prepare summary
+    summary = {
+        'available_transforms': available_transforms,
+        'unique_atlases': sorted(list(atlases)),
+        'direct_transforms': len(direct_transforms),
+        'all_possible_paths': all_paths,
+        'total_possible_paths': len(all_paths)
+    }
+    
+    return summary
+
+def print_transform_summary(summary: dict):
+    """Print a formatted summary of transform search results."""
+    print("=" * 60)
+    print("BRAIN ATLAS TRANSFORM SEARCH RESULTS")
+    print("=" * 60)
+    
+    print(f"\nUnique Atlases Found: {len(summary['unique_atlases'])}")
+    print(", ".join(summary['unique_atlases']))
+    
+    print(f"\nDirect Transforms Available: {summary['direct_transforms']}")
+    
+    print(f"\nTotal Possible Paths: {summary['total_possible_paths']}")
+    
+    print("\nDirect Paths:")
+    direct_paths = {k: v for k, v in summary['all_possible_paths'].items() if v['direct']}
+    for path_name, path_info in direct_paths.items():
+        print(f"  {path_name}")
+    
+    print("\nIndirect Paths (multi-step):")
+    indirect_paths = {k: v for k, v in summary['all_possible_paths'].items() if not v['direct']}
+    for path_name, path_info in indirect_paths.items():
+        path_str = " -> ".join(path_info['path'])
+        print(f"  {path_name} (steps: {path_info['length']})")
+        print(f"    Route: {path_str}")
+    
+    print("\nAvailable Transform Files:")
+    for i, transform in enumerate(summary['available_transforms'], 1):
+        print(f"  {i}. {transform.source} -> {transform.target}")
+        print(f"     Density: {transform.density}, "
+              f"Hemisphere: {transform.hemisphere}, "
+              f"Resource: {transform.resource_name}")
+
```

Comment:
This seems like it would be useful for debugging. I wonder if it makes sense to break this down into smaller pieces that can be called individually?